llvm/lib/Transforms/IPO/AttributorAttributes.cpp

```diff
@@ -5220,6 +5220,13 @@ static unsigned getKnownAlignForUse(Attributor &A, AAAlign &QueryingAA,
         return AlignAA->getKnownAlign().value();
       break;
     }
+    case Intrinsic::amdgcn_make_buffer_rsrc: {
```

Comment:
add a comment here regarding the handling of target specific handling and some future direction that we might take.